api/tests/routes/pro/signup_pro_test.py

```diff
@@ -5,7 +5,7 @@
 import pcapi.core.offerers.models as offerers_models
 from pcapi.core.offerers.models import Offerer
 from pcapi.core.offerers.models import UserOfferer
-from pcapi.core.users.factories import ProFactory
+import pcapi.core.users.factories as users_factories
```

Comment:
Par homogénéité avec ce qu'on essaie de faire globalement dans le backend depuis une discussion que je ne saurais retrouver : 
```suggestion
from pcapi.core.users import factories as users_factories
```

Reply:
Je crois que c'était le sujet d'une des premières synchros backend que j'ai connue, donc ça a presque un an 😁

Reply:
Modifié, mais petite question, pourquoi c'est pas fait d'un coup sur l'ensemble du projet?